### PR TITLE
Fix show meta

### PR DESCRIPTION
**Pull request by snewcomer**
This is my fault.  I was trying to figure out for the longest time why the `show` meta was not working properly.  Looking for a test to write to back this up.

Improves on https://github.com/jeregrine/jsonapi/pull/97

### CODE REVIEW

**Comment by snewcomer** [comment]
It would be great to add some tests around this [behavior](https://github.com/jeregrine/jsonapi/blob/master/lib/jsonapi/view.ex#L152).  Not seeing anything yet.

**Comment by snewcomer** [comment]
Added some tests.  Although a bit mystified on the failures on 1.6.  It seems something to do with the formatter, but I may be missing some knowledge of what to do.

**Comment by jeregrine** [comment]
I believe you just need to run the formatter on the test file.

**Comment by snewcomer** [comment]
Ah thank you.  I wasn't on 1.6 previously!  So one other problem is the console output.  `Redefining module JSONAPI.ViewTest.CommentView (current version defined in memory)` && `redefining module Phoenix...`

I'm wondering how we can load Phoenix once for the CommentView in the test.  Any thoughts?  Maybe a test helper macro?

https://travis-ci.org/jeregrine/jsonapi/jobs/349849195

**Comment by jeregrine** [comment]
I wonder if setup tags is even required.

**Comment by snewcomer** [comment]
@jeregrine That is a good idea.  So if I remove `setup tags`, I could also remove [this](https://github.com/jeregrine/jsonapi/blob/master/lib/jsonapi/view.ex#L143) and all tests pass; however, I am also lacking context on why that is needed and what various use cases `ensure_loaded` covers.

**Comment by snewcomer** [comment]
@jeregrine So I added Phoenix as a test dep.  Lmk what you think.  Also tested the render functions as well.

File: lib/jsonapi/view.ex [hunk 1]
```diff
@@ -140,12 +140,12 @@ defmodule JSONAPI.View do
         "#{url_for(data, conn)}/relationships/#{rel_type}"
       end
 
-      if Code.ensure_loaded?(Phoenix) do
+      with true <- Code.ensure_loaded?(Phoenix) do
```
Comment (doomspork) [inline lib/jsonapi/view.ex]: Why `with` over an `if` here?  I'm not sure we gain anything.

Comment (snewcomer) [reply lib/jsonapi/view.ex]: @doomspork Added `raise ArgumentError` in the `else` case.  Should we keep that or remove it?  Don't mind either way, but thought, in a very rare case, might help somebody.

**Comment by doomspork** [comment]
@snewcomer any updates here?

**Comment by snewcomer** [comment]
@jeregrine @doomspork This PR is ready for your re-review!

**Comment by doomspork** [comment]
@jeregrine I do want to merge with your review still showing rejected.  Do you want to dismiss or re-review?

**Comment by doomspork** [comment]
Thank you @jeregrine ❤️

**Comment by jeregrine** [comment]
Thank you all! I will try to cut release tonight.

**Pull request merged**